docs/en/Running-Spark-on-Alluxio.md

```diff
@@ -73,6 +73,19 @@ Alternatively you can add the properties to the previously created Hadoop config
 </configuration>
 ```
 
+## Check Spark with Alluxio integration
+
+Before running Spark on Alluxio, you might want to make sure that your Spark configurations has been 
+setup correctly for integrating with Alluxio. The Spark integration checker can help you achieve this. 
+
+When you have a running Spark cluster (or Spark standalone), you can run the following command in the Alluxio project directory:
+
+```bash
+$ checker/bin/alluxio-checker.sh Spark <spark master uri> <spark partition number(optional)>
```

Comment:
does it require `Spark` or `spark` is fine?

Reply:
spark is fine, I will change that